browser-app.cpp

```diff
@@ -130,11 +130,61 @@ void BrowserApp::OnContextCreated(CefRefPtr<CefBrowser> browser,
 	obsStudioObj->SetValue("getStatus", getStatus,
 			       V8_PROPERTY_ATTRIBUTE_NONE);
 
+	CefRefPtr<CefV8Value> startRecording =
```

Comment:
As a side note, before I review/merge this PR, I'll first be sure to submit my commit to simplify this code, which'll help this implementation significantly.

It'll become something like this instead:

```cpp
	std::vector<std::string> functions = {"getCurrentScene", "getStatus", "saveReplayBuffer"};

	for (std::string name : functions) {
		CefRefPtr<CefV8Value> func =
			CefV8Value::CreateFunction(name, this);
		obsStudioObj->SetValue(name, func, V8_PROPERTY_ATTRIBUTE_NONE);
	}